generate-type-definitions.js

```diff
@@ -3,12 +3,7 @@
 const { execSync } = require('child_process');
 const { readFileSync, writeFileSync } = require('fs');
 
-try {
-	execSync('tsc -p src/compiler --emitDeclarationOnly && tsc -p src/runtime --emitDeclarationOnly');
-} catch (err) {
-	console.error(err.stderr.toString());
-	throw err;
-}
+execSync('tsc -p src/compiler --emitDeclarationOnly && tsc -p src/runtime --emitDeclarationOnly', { stdio: 'inherit' });
```

Comment:
It's a drive-by fix, that way the output from `execSync` is inlined into the main output and we get proper console error output. If we don't do this, we get gibberish whenever a type check error occurs instead of seing the error.